src/main/java/org/fcrepo/spec/testsuite/crud/Container.java

```diff
@@ -138,12 +139,23 @@ public void ldpcContainmentTriples() {
         final Response container = createBasicContainer(getLocation(base), "container");
         final Response containerChild = createBasicContainer(getLocation(container), "child");
 
-        final Response direct = createDirectContainer(
-                getLocation(base), DIRECT_CONTAINER_BODY.replace("%membershipResource%", getLocation(container)));
-        final Response directMember = createBasicContainer(getLocation(direct), "member");
+        // Skips adding a direct container and the triple check, if not supported
+        boolean directContainerTripleCheck = true;
+        Response directMember = null;
+        try {
+            skipIfDirectContainersNotSupported();
+            final Response direct = createDirectContainer(
+                    getLocation(base), DIRECT_CONTAINER_BODY.replace("%membershipResource%", getLocation(container)));
+            directMember = createBasicContainer(getLocation(direct), "member");
+        } catch (SkipException e) { // DC not supported
+            directContainerTripleCheck = false;
+        }
 
+        // Only omit, not include, can guarantee the omission of specified triples.
         final Response resP = doGet(getLocation(container),
-                new Header("Prefer", "return=representation; include=\"http://www.w3.org/ns/ldp#PreferContainment\""));
+                new Header("Prefer", "return=representation; " +
```

Comment:
This line sets a Prefer headers that omits minimal and membership triples. Using "Omit" is the only way you can guarantee that a particular triple set is not present. Fedora doesn't seem to honor this currently, so I think we've identified a compliance gap w/LDP here.

Reply:
Hi @gregjan - I believe you're correct and have filed this bug accordingly: https://jira.duraspace.org/browse/FCREPO-3037